target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -29,6 +29,11 @@ xiaomi,redmi-router-ax6000-stock|\
 xiaomi,redmi-router-ax6000-ubootmod)
 	ucidef_set_led_netdev "wan" "wan" "rgb:network" "wan"
 	;;
+zbtlink,zbt-z8103ax)
+	ucidef_set_led_netdev "wan" "wan" "green:wan" "eth1" "link tx rx"
+	ucidef_set_led_netdev "wlan2g" "wlan2g" "green:wlan2g" "phy0-ap0"
+	ucidef_set_led_netdev "wlan5g" "wlan5g" "green:wlan5g" "phy1-ap0"
```

Comment:
You can use dts `linux,default-trigger` for the wireless leds. The dts is more preferable way than the script. Here is an example:
https://github.com/openwrt/openwrt/blob/main/target/linux/ramips/dts/mt7621_tplink_ec330-g5u-v1.dts#L76
https://github.com/openwrt/openwrt/blob/main/target/linux/ramips/dts/mt7621_tplink_ec330-g5u-v1.dts#L85

Reply:
Ohhh, but what is the correct value should I put to it? 
Will this work? `linux,default-trigger = "phy0-ap0";`

Reply:
Please try `linux,default-trigger = "phy0tpt";` for phy0 and `linux,default-trigger = "phy1tpt";` for phy1 (like in example).

Reply:
Thank you it worked :bow: :partying_face: